src/background/sidebar-injector.js

```diff
@@ -143,6 +145,12 @@ export function SidebarInjector() {
     return url.startsWith('https://bookshelf.vitalsource.com/');
   }
 
+  /** @param {string} url */
+  function isLMSAssignmentURL(url) {
+    const { origin } = new URL(url);
+    return origin.includes('lms.hypothes.is');
```

Comment:
This won't work for the Canada LMS, whose URL is `lms.ca.hypothes.is`. I would suggest to do something like:

1. Check that the TLD is `hypothes.is` (ie. host ends with `.hypothes.is`)
2. Check for the token `lms` in the subdomains (eg. by tokenizing into alphanumeric tokens)